ducks/appVersionList.ts

```diff
@@ -0,0 +1,33 @@
+import { Dispatch, combineReducers } from 'redux';
+import { createAction, createReducer } from 'deox';
+
+import api from '@/services/ship-api';
+import { actionTypeCreator } from '@/utils';
+import { AppVersionList } from '@/models';
+
+const $ = actionTypeCreator('APP_VERSION');
```

Comment:
Please use a unique prefix
```suggestion
const $ = actionTypeCreator('APP_VERSION_LIST');
```